### DISCUSSION TITLE
Add Wind Gust Speed to Weather platform

### DISCUSSION THREAD

**Discussion by gjohansson-ST** [home-assistant/architecture#778] [Entity Models] [answered]
Many/most weather providers also provides wind gust in addition to the average wind speed.
The unit of measurements would be same and follow the implemented logic of user options.

Suggestion: Implement wind gust in addition to the existing wind speed using the logic already in place.

**Comment by emontnemery** [comment]
I think this would be a useful addition:
- Integrations `climacell` and `smhi` already expose wind gust as extra attributes on the weather entity.
- Integrations `accuweather`, `aemet`, `buienradar`, `dark_sky`, `environment_canada`, `meteo_france`, `metoffice`, `nws`, `tomorrowio` expose wind gust sensors.

**Reply by jeeftor** [reply]
Would this just be for the weather or the forecast portion?

**Comment by frenck** [comment, marked as answer]
We have discussed this architectural proposal and are in agreement this property can be added to both the weather entity and the forecast model of the weather entity.

../Frenck

**Comment by gjohansson-ST** [comment]
PR opened: https://github.com/home-assistant/core/pull/95065

**Reply by gjohansson-ST** [reply]
Implemented so will close this

**Reply by jeeftor** [reply]
woohoo
